java/BUILD

```diff
@@ -2,6 +2,11 @@ package(default_visibility = ["//visibility:public"])
 
 load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build")
 load("@package_bundle//file:packages.bzl", "packages")
+load("//cacerts:java.bzl", "cacerts_java")
+
+cacerts_java(
+    name = "cacerts_java.tar",
```

Comment:
Given that `out` is `%{name}.tar` I think the `tar` here is redundant because the actual `tar` is going to get named `cacerts_java.tar.tar` :)